src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -363,10 +385,13 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
         // Authentication listeners
         list($authListeners, $defaultEntryPoint) = $this->createAuthenticationListeners($container, $id, $firewall, $authenticationProviders, $defaultProvider, $configuredEntryPoint);
 
+        $config->replaceArgument(6, $configuredEntryPoint ?: $defaultEntryPoint);
```

Comment:
> we should probably not expose it but instead simply list enabled listeners for the given firewall. Thus you can simply show in the profiler the list of enabled listeners.

I think you're right. Some sensible data such as passphrase, secret (i.e. anonymous), ... should probably not be displayed in the profiler (#19490).

Would you totally avoid storing the configuration of each registered listener and only store their names instead? Or keep it as is and just list `array_keys($listenerConfigs)` in #19490?